sys/sys/capsicum.h

```diff
@@ -40,6 +40,7 @@
 #define	_SYS_CAPSICUM_H_
 
 #include <sys/param.h>
+#include <sys/types.h>
```

Comment:
This should likely just be added to `./tools/build/cross-build/include/linux/sys/param.h`